beginner/cogs/buddy.py

```diff
@@ -142,6 +144,111 @@ async def archive_buddy_chat(self, ctx: nextcord.ext.commands.Context):
         await ctx.channel.edit(name=f"{ctx.channel.name}-archive")
         await ctx.channel.send("🗂 This channel has been archived")
 
+    @Cog.slash_command(
+        name="look-for-buddy",
+        description="Look for a buddy",
+    )
+    async def look_for_buddy(self, interaction: nextcord.Interaction):
+        await interaction.response.send_modal(LookForBuddy())
+
+
+class LookForBuddy(nextcord.ui.Modal):
+    def __init__(self):
+        super().__init__(
+            title="Look For Buddy",
```

Comment:
Can this be changed to Looking For Buddy please?

Reply:
Yup!